packages/jest-changed-files/src/__tests__/hg-test.js

```diff
@@ -32,13 +32,13 @@ describe('hgChecker', () => {
 
   describe('isHGRepository', () => {
 
-    pit('returns null for non hg repo folder', () => {
-      return hg.isHGRepository(tmpdir).then(res => {
+    it('returns null for non hg repo folder', () =>
+      hg.isHGRepository(tmpdir).then(res => {
```

Comment:
yes we are, it's an arrow function:

`it('tests a promise', () => foo.then(…))`